### DISCUSSION TITLE
No Sandbox Folder on iPhone

### DISCUSSION THREAD

**Discussion by clef-shanty** [MobiusSync/MobiusSync#117] [Beta]
I've installed the Mobius Sync app three times on my iPhone 14 pro max running iOS ver 16.4.1 (a) and there is no Sandbox folder installed on the phone for the App. What gives? How do I fix this? Are there better instructions for this app?

**Comment by SimonPickup** [comment]
Please try rebooting iOS. The folder often seems to appear missing until a reboot.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Clef Shanty ***@***.***>
Sent: Thursday, May 18, 2023 3:21:33 PM
To: MobiusSync/MobiusSync ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [MobiusSync/MobiusSync] No Sandbox Folder on iPhone (Issue #115)


I've installed the Mobius Sync app three times on my iPhone 14 pro max running iOS ver 16.4.1 (a) and there is no Sandbox folder installed on the phone for the App. What gives? How do I fix this? Are there better instructions for this app?

**Reply by clef-shanty** [reply]
That worked. Thank you.

**Comment by sai-foss** [comment]
thank you so much @sjpickup . i thought i was going crazy... maybe a mention of this in the faq would be nice 👍
